meta/CSVPageImporter.php

```diff
@@ -7,6 +7,9 @@ class CSVPageImporter extends CSVImporter {
 
     protected $importedPids = array();
 
+    /** @var bool  */
+    protected $createPage = [];
```

Comment:
What is it? `bool` or an `array`? Maybe `bool[]`?

Reply:
Started as bool, became an array of bool.
Fixed.